src/main/ArrayVisualizer.java

```diff
@@ -321,6 +379,55 @@ public void run() {
         this.arrays = new ArrayList<>();
         this.arrays.add(this.array);
 
+        this.fontSelection = "Times New Roman";
+        this.fontSelectionScale = 25;
+        List<Integer> statsInfoList = new ArrayList<>();
+        try (Scanner statsScanner = new Scanner(new File("stats-config.txt"))) {
+            while (statsScanner.hasNextLine()) {
+                String line = statsScanner.nextLine().trim();
+                if (line.length() > 0 && line.charAt(0) == '#') continue;
+                if (line.startsWith("FONT: ")) {
+                    String font = line.substring(5);
+                    int fontScale = 25;
+                    int starIndex;
+                    if ((starIndex = font.indexOf('*')) != -1) {
+                        fontScale = Integer.parseInt(font.substring(starIndex + 1).trim());
+                        font = font.substring(0, starIndex);
+                    }
+                    fontSelection = font.trim();
+                    fontSelectionScale = fontScale;
+                    continue;
+                }
```

Comment:
what about requiring that this is the first line?

Reply:
Because I don't wanna require that

Reply:
could have another loop before, but yeah I guess it doesn't hurt